cmd/lifecycle/exporter.go

```diff
@@ -109,6 +110,13 @@ func (e *exportCmd) Exec() error {
 		cmd.Logger.Infof("no stack metadata found at path '%s', stack metadata will not be exported\n", e.stackPath)
 	}
 
+	projectMetadataPath := path.Join(".", lifecycle.ProjectMetadataFileName)
+	var projectMD lifecycle.ProjectMetadata
+	_, err = toml.DecodeFile(projectMetadataPath, &projectMD)
+	if err != nil {
```

Comment:
We should use `os.IsNotExist` to distinguish between missing project metadata and another type of error and fail in the latter case.